automation/consent-automation-config.yml

```diff
@@ -0,0 +1,52 @@
+server:
+  type: simple
+  applicationContextPath: /
+  adminContextPath: /admin
+  connector:
+    type: http
+    port: 8000
+database:
+  driverClass: org.postgresql.Driver
+  user: consent
+  password: consent
+  url: jdbc:postgresql://consent-postgres:5432/consent
+  initialSize: 20
+  minSize: 20
+  maxSize: 128
+  validationQuery: SELECT 1
+googleStore:
+  password: password
+  bucket: bucket
+  endpoint: endpoint
+# Logging settings
+logging:
+  # default logging level
+  level: WARN
+  # logger specific levels
+  loggers:
+    "org.broadinstitute.consent.http.cloudstore.GCSService": OFF
```

Comment:
Might be worth setting this to debug and removing the custom loggers. We can dial it back if the logs get too crazy.